content/utilities/map-export/plugin.js

```diff
@@ -68,6 +61,37 @@ class Plugin {
       this.status.style.color = 'red';
     }
   }
+  
+  onImport = async () => {
+    let input;
+    try {
+      input = await window.navigator.clipboard.readText();
+      this.processMap(input);
+    } catch (err) {
+      console.error(err);
+      this.status.innerText = 'Unable to import map. Did you allow clipboard access?';
+      this.status.style.color = 'red';
+      return;
+    }
+  }
+  
+  onUpload = async () => {
+    let input;
+    try {
+      var file = inputFile.files.item(0);
+      var reader = new FileReader();
+      reader.onload = function() {
+        input = reader.result;
+        this.processMap(input);
+      }.bind(this);
```

Comment:
```suggestion
    try {
      var file = inputFile.files.item(0);
      var reader = new FileReader();
      reader.onload = () => {
        this.processMap(reader.result);
      };
```
light cleanup/streamline